oom-frontend/src/app/shared/modules/maintainer/maintainer-editor/maintainer-editor.component.ts

```diff
@@ -0,0 +1,29 @@
+import {Component, EventEmitter, Input, OnInit, Output} from '@angular/core';
+import {User} from "../../../models/user.model";
+
+@Component({
+  selector: 'maintainer-editor',
+  templateUrl: './maintainer-editor.component.html',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Out-Of-Memory-Team/out-of-memory/pullRequest?prid=7533382)
```suggestion
  templateUrl: "./maintainer-editor.component.html",
```